web/sites/default/config/webform.webform.report_campsite.yml

```diff
@@ -707,10 +706,10 @@ handlers:
       requester_email: contact_email
       subject: 'Campsite Report Inside Park'
       comment: "A campsite has been reported inside of a park by a community member:\r\n\r\n[webform_submission:values:html]"
-      tags: 'drupal webform campsite'
+      tags: 'drupal webform campsite campsite_in_park'
       priority: normal
       status: new
-      group_id: '16341182522007'
+      group_id: '12927681093399'
```

Comment:
This changes the group for tickets assigned to the Park Rangers from Ranger Supervisors to Ranger Dispatch. It does not affect the campsite report schema or data export in any way.